README.md

```diff
@@ -48,6 +63,54 @@ sudo make install
 
 ## Tips
 
+### Control blinking LEDs with `it87`
+
+*Note:* This is probably not supported on all devices, only ones that use the IT8625E or IT8720F
+chips or similar.
```

Comment:
At least I don't see any behavioral changes on my AS61 with `IT8728F`, so I don't think it's supported on there.

Btw, does this work or conflict with setting a trigger for the led? E.g. `echo cpu | sudo tee /sys/class/leds/green:status/trigger`?

Reply:
This doesn't seem to work together.
Even when setting the LED to "always on", the trigger doesn't do anything.
And if I set gpled1_blink to `0`, I still can't configure the green:status LED (at all, neither trigger nor brightness).
No idea why, maybe additionally some bit must be set somewhere so an LED can be controlled freely, if it has been controlled by that blinking feature first

Reply:
good news: I found a way to make blinking control work more properly!
I found the reg and bits that must be set to switch the GPIO between "Simple I/O mode" which lets you use brightness and trigger etc, and "Alternate mode" which lets you use that builtin blinking feature

Reply:
That's really cool, nice work! I can now use the `gpled` to blink any led on my AS61 too.

It behaves a bit weirdly for the `green:status` led (seems to be a long pause after two blink intervals or something, but on other LEDs it is consistent).